devel/osslsigncode/Portfile

```diff
@@ -36,3 +36,16 @@ depends_lib         port:curl \
                     path:lib/libssl.dylib:openssl
 
 livecheck.regex     /${name}-(\[0-9.\]+)${extract.suffix}
+
+subport osslsigncode-devel {
+    version             1.8
```

Comment:
Since you are actually using a git branch, you should probably change the version to add the date of the branch so you can update it easily (would the next update still be version 1.8?) and people know what they are getting. So change it to something like this, depending on the date:

`    version             1.8-20190424
`

Reply:
Using:

`version 1.8-20150831`

Makes sense to me.